jenkins/Jenkinsfile

```diff
@@ -0,0 +1,256 @@
+            container('py36'){
+              dir("${env.WORKSPACE}/flake8") {
+              echo 'install flake 8'
+              sh "pip install flake8 --user"
+              echo 'run flake8'
+              sh "python -m flake8 dsl_parser script_runner cloudify cloudify_rest_client"
+              }
+            }
+          }
+        }
+      }
+    }
+    stage('tests'){
+      parallel{
+        stage('test_py26'){
+          steps{
+            sh "mkdir -p ${env.WORKSPACE}/test-py26 && cp -rf ${env.WORKSPACE}/${env.PROJECT}/. ${env.WORKSPACE}/test-py26"
+            container('py26'){
+              dir("${env.WORKSPACE}/test-py26") {
+                echo 'setup_py26 - Install and set python version with pyenv'
+                sh '''
+                #!/bin/bash
+                set -eux
+                git clone https://github.com/yyuu/pyenv.git ~/.pyenv
+                export PYENV_ROOT="$HOME/.pyenv"
+                export PATH="$PYENV_ROOT/bin:$PATH"
+                echo 'export PATH="$HOME/.pyenv/bin:$PATH"' >> ~/.bashrc
+                sudo apt-get install -y build-essential libssl1.0-dev zlib1g-dev xz-utils
+                echo 'if [ -e $SHLVL = 1 ]; then
+                  echo 'eval "$(pyenv init -)"' >> ~/.bashrc
+                  echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.bashrc
+                  fi'
+                pyenv install 2.6.9
+                pyenv global 2.6.9
+                '''     
+                install_test_dependencies('py26')
+                pytest('dsl_parser')
+                pytest('script_runner')
+                pytest('cloudify')
+                /* store test results in path: test-results
+                */
+              }
+            }
+          }
+        }
+        stage('test_py27'){
+          steps{
+            sh "mkdir -p ${env.WORKSPACE}/test-py27 && cp -rf ${env.WORKSPACE}/${env.PROJECT}/. ${env.WORKSPACE}/test-py27"
+            container('py27'){
+              dir("${env.WORKSPACE}/test-py27") {
+                install_test_dependencies('py27')
+                pytest('dsl_parser')
+                pytest('script_runner')
+                pytest('cloudify')
+                /* storte test results in path: test-results */
+              }
+            }
+          }
+        }
+        stage('test_py36'){
+          steps{
+            sh "mkdir -p ${env.WORKSPACE}/test-py36 && cp -rf ${env.WORKSPACE}/${env.PROJECT}/. ${env.WORKSPACE}/test-py36"
+            container('py36'){
+              dir("${env.WORKSPACE}/test-py36") {
+                install_test_dependencies('py36')
+                pytest('dsl_parser')
+                pytest('script_runner')
+                pytest('cloudify')
+                /* store test results in path: test-results */
+              }
+            }
+          }
+        }
+      }
+    }
+    stage('build_all_project_docs'){
```

Comment:
the step looks really bad inside the jenkins, please add `echo` with pretty messaging
https://jenkins.eks.cloudify.co/blue/organizations/jenkins/Common-Multibranch/detail/yuval-build/3/pipeline/167